test/articleTest.js

```diff
@@ -251,6 +251,28 @@ describe('article like', () => {
   });
 });
 
+describe('article update', () => {
+  it('should update an article', async () => {
+    const result = await server.patch(`/api/v1/articles/${articleId}`)
+      .set('authorization', testToken)
+      .send({
+        body: 'it is not good',
+        plainText: 'jsjfosdf'
+      }).expect(201);
+    expect(result.status).to.equal(201);
+    expect(result.body.article.body).to.equal('it is not good');
+    expect(result.body.article.plainText).to.equal('jsjfosdf');
+  });
+  it('should update an article', async () => {
+    const result = await server.patch(`/api/v1/articles/${articleId}`)
+      .set('authorization', testToken)
+      .send({
+        body: 'it is not good',
+        plainText: 'jsjfosdf'
+      }).expect(201);
+    expect(result.status).to.equal(201);
+  });
+});
```

Comment:
Hey, good job on capturing the edge cases in the test, but kindly look into the *it* blocks as one is a duplicate of the other.

Reply:
would do so ..thanks